lib/src/chart/radar_chart/radar_chart_data.dart

```diff
@@ -154,6 +155,9 @@ class RadarChartData extends BaseChartData with EquatableMixin {
   /// Handles touch behaviors and responses.
   final RadarTouchData radarTouchData;
 
+  /// If [isMinValueAtCenter] is true, the minimum value of the [RadarChart] will be at the center of the chart.
+  final bool isMinValueAtCenter;
```

Comment:
You need to add this property in the `copyWith()`, `lerp()` and the `List<Object?> get props => []` functions